### ISSUE TITLE
PL2303GC

### ISSUE THREAD

**Issue by Vlvic** [felHR85/UsbSerial#354]
Will the library work with:
usb-device vendor-id="1659" product-id="9123" /> <!-- 0x23a3: PL2303GC 
VID_067B&PID_23A3

**Comment by apichaiterm** [comment]
I can't use it . 
UsbDevice: /dev/bus/usb/002/003, 67b-23a3 USB-Serial Controller , Prolific Technology Inc.  DEBMb115818
 [ERROR:flutter/runtime/dart_vm_initializer.cc(41)] Unhandled Exception: PlatformException(UsbSerialPortAdapter, Not an Serial device., null, null)
 
How do I fix?

**Comment by realrkace** [comment]
No, you will need to add the PID/VID in the list in the java files, recompile the aar and then use it in your project, best way is to fork or create a pull request. This works with the listed PL2302 chips in the java files, but not other equipment. There should be a compatibility layer that allows connecting to any type of device rather than limit to known PIDs/VIDs
